pkg/plugin/resource_handlers.go

```diff
@@ -0,0 +1,22 @@
+package plugin
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// Handlers stores the list of http.HandlerFunc functions for the different resource calls
+type Handlers struct {
+	Labels     http.HandlerFunc
+	Milestones http.HandlerFunc
+}
+
+// GetRouter creates the gorilla/mux router for the HTTP handlers
+func GetRouter(h Handlers) *mux.Router {
+	router := mux.NewRouter()
+	router.Path("/labels").Methods("GET").HandlerFunc(h.Labels)
+	router.Path("/milestones").Methods("GET").HandlerFunc(h.Milestones)
+
+	return router
+}
```

Comment:
```suggestion
type Handlers struct {
	Labels     http.HandlerFunc
}

// GetRouter creates the gorilla/mux router for the HTTP handlers
func GetRouter(h Handlers) *mux.Router {
	router := mux.NewRouter()
	router.Path("/labels").Methods("GET").HandlerFunc(h.Labels)

	return router
}
```

The `Milestones` thing is not being used, so I'd remove it meanwhile.